nntrainer/graph/network_graph.cpp

```diff
@@ -404,6 +436,16 @@ LayerNode &NetworkGraph::getLayerNode(const std::string &layer_name) {
   throw std::invalid_argument("Cannot find Layer");
 }
 
+LayerNode &NetworkGraph::getSortedLayerNode(const std::string &layer_name) {
```

Comment:
Q. is this different from `getLayerNode(const std::string &layer_name)` ?

Reply:
different. but we could merge. getLayerNode get the layer from layers, but getSortedLayerNode get the layer form Sorted which includes layers which does not exist in layers.

Reply:
I think it is the same: just for the getLayerNode with layer_name. if it is idx, then it differ. I will fix.